mod-linked-data/src/main/resources/citation/mod-linked-data/features/suppress-flags/suppress-flags.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Suppress Flags in Inventory
+
+  Background:
+    * url baseUrl
+    * call login testUser
+    * def testUserHeaders = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*' }
+    * configure headers = testUserHeaders
+
+  Scenario: Create an instance in Inventory, change Suppress Flags and check their values in search index
+    # Create a new instance in Inventory
+    * def workRequest = read('samples/create-work-request.json')
+    * def postWorkCall = call postResource { resourceRequest: '#(workRequest)' }
+    * def workId = postWorkCall.response.resource['http://bibfra.me/vocab/lite/Work'].id
+    * def resourceRequest = read('samples/create-instance-request.json')
+    * call postResource
+
+    # Search for the new instance in Inventory's search
+    * def query = 'title all "suppress_flags"'
+    * def inventoryInstanceSearchResponse = call searchInventoryInstance
+    * def instanceSearch = inventoryInstanceSearchResponse.response.instances[0]
+    * def inventoryInstanceId = instanceSearch.id
+
+    # Update suppress flags
+    * def getInventoryInstanceResponse = call getInventoryInstance { id: "#(inventoryInstanceId)" }
+    * def inventoryInstance = getInventoryInstanceResponse.response
+    * eval inventoryInstance['staffSuppress'] = true
+    * eval inventoryInstance['discoverySuppress'] = true
+    * call putInventoryInstance
+
+    # Search the Check Suppress Flags values
+    Given path 'search/linked-data/works'
+    And param query = query
+    And retry until response.content[0].instances[0].suppress.staff == true && response.content[0].instances[0].suppress.fromDiscovery == true
+    When method GET
```

Comment:
When I executed this test in my local, it failed with the following error
```
17:02:01.825 retry condition satisfied
17:02:01.827 classpath:citation/mod-linked-data/features/suppress-flags/suppress-flags.feature:34
When method GET
js failed:
>>>>
01: response.content[0].instances[0].suppress.staff == true && response.content[0].instances[0].suppress.fromDiscovery == true
<<<<
org.graalvm.polyglot.PolyglotException: TypeError: Cannot read property "staff" from undefined
- <js>.:program(Unnamed:1)

classpath:citation/mod-linked-data/features/suppress-flags/suppress-flags.feature:34
```

Are you getting any failure?

Reply:
No @pkjacob, i'm not, all tests are green:
<img width="500" alt="image" src="https://github.com/user-attachments/assets/82ad2404-6475-4efa-8929-91a21b519fba">
<img width="500" alt="image" src="https://github.com/user-attachments/assets/0d16267c-f7cb-4131-b136-abaed994d654">

Reply:
it's also green with the changes in the work sample file mentioned above.